generative_ai/prompts/test_prompt_template.py

```diff
@@ -11,10 +11,60 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from vertexai.preview import prompts
 
+
+import prompt_create_save_generate
+import prompt_delete
+import prompt_list_prompts
+import prompt_list_version
+import prompt_load_or_retrieve
+import prompt_restore_version
 import prompt_template
 
 
 def test_prompt_template() -> None:
     text = prompt_template.prompt_template_example()
     assert len(text) > 2
+
+
+def test_prompt_create_save_generate() -> None:
+    response = prompt_create_save_generate.prompt_local_prompt_create()
+    assert response
+
+
+def test_prompt_list_prompts() -> None:
+    list_prompts = prompt_list_prompts.list_prompt()
+    assert list_prompts
+
+
+def test_prompt_load_or_retrieve() -> str:
+    get_prompt = prompt_load_or_retrieve.get_prompt()
+    assert get_prompt
+
+
+def test_prompt_list_version() -> None:
+    list_versions = prompt_list_version.list_prompt_version()
+    assert list_versions
+
+
+def test_prompt_delete() -> None:
+    prompt = prompts.Prompt(
+        prompt_name="Test-prompt",
+        prompt_data="This is a test prompt.",
+    )
+
+    prompt1 = prompts.create_version(prompt=prompt)
+    prompt_id = prompt1.prompt_id
+    prompt_delete.delete_prompt()
```

Comment:
Trying to understand this test. So this test creates a Prompt object locally, and creates a version for it.

Then prompt_delete.delete_prompt() is called, which is your snippet above. In your snippet, you make a **_second_** Prompt object locally, create a version for it, and then delete it.

But the test expects the **_first_** Prompt resource to be deleted. 

Is this test successfully passing when you run it?

Reply:
This was left in when doing local testing. Apparently I didn't push up corrected version of this file...Resolving now.